docs/conf.py

```diff
@@ -47,7 +47,7 @@
 
 # General information about the project.
 project = u'Invenio-Previewer'
-copyright = u'2016, CERN'
```

Comment:
Can you remove the mention to the license in the [`README.rst`](https://raw.githubusercontent.com/inveniosoftware/invenio-previewer/master/README.rst)? Lets align it with other modules READMEs such as [Invenio-Records](https://invenio-records.readthedocs.io/en/latest/):
```
Further documentation available Documentation: https://invenio-previewer.readthedocs.io/
```